site/topics/problems/multi-objective.rst

```diff
@@ -44,13 +44,41 @@ Multi-Objective Problems
 Scalarization
 =============
 
-Distance from Ideal
--------------------
+* Scalarization is the process of *scaling* the various features to be optimized and combining them into a single value
+
+    * It is a way to turn multi-objective problems into a single objective problem
+
+
+* It is an *a priori* process, meaning the decisions about the scaling must be done before the search starts
 
 
 Weighted Sum
 ------------
 
+* The simplest strategy is a *weighted sum*
+* Scale each individual objective being optimized and add them together to make a single value
+
+    :math:`w_{1}f_{1}(x) + w_{2}f_{2}(x) + w_{3}f_{3}(x) + ... + w_{m}f_{m}(x)`
+
+    Or
+
+    :math:`\sum_{i}^{m}w_{i}f_{i}(x)`
+
+
+* Where
+
+    * :math:`x` is some chromosome
+    * :math:`m` is the number of objectives being optimized
+    * :math:`f_{i}(x)` is a function returning the :math:`i^{th}` objective's fitness on chromosome :math:`x`
+    * :math:`w_i` is the weight/scale of objective :math:`i`
+
+
+* If one objective is more important than another, assign it a stronger weight
+* Use negative weights to account for objectives that are to be minimized/maximized
+
+* Unfortunately, however, weighted sums do not work too well beyond two or three objectives
```

Comment:
Not GP, but I've done this at work with hundreds to thousands of objectives. Works good enough.